Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -72,6 +72,16 @@ std::string VolumeGC::GetGameID(const Partition& partition) const
   return DecodeString(id);
 }
 
+std::string VolumeGC::GetGameTDBID(const Partition& partition) const
+{
+  const std::string game_id = GetGameID(partition);
+
+  if (game_id == "GNHE5d" && !GetBootDOLOffset(*this, partition).has_value())
+    return "";  // This is one of the Datel discs that are using the game ID of NHL Hitz 2002
```

Comment:
This comment should probably go on top of the if statement to make it nicer if refactoring ever happens here for whatever reason.